app/app.go

```diff
@@ -93,12 +93,13 @@ func (a *App) Start(ctx context.Context) error {
 	}
 	a.Run("FolderWatcher", watcher)
 
+	var ipfsNode *node.IpfsNode
 	// setup local ipfs node if Ipfsnode is set
 	if a.cfg.GetBool(config.Ipfsnode, true) {
 		// setup local ipfs node
-		node := node.NewIpsNode(a.cfg)
-		err = a.RunAsync("IpfsNode", node, func() error {
-			return node.Start(ctx)
+		ipfsNode = node.NewIpsNode(a.cfg)
```

Comment:
I think it would be better to do this:
```suggestion
		ipfsNode := node.NewIpsNode(a.cfg)
```

and delete the 
```
var ipfsNode *node.IpfsNode
```

Reply:
Fixed in [9bc2f71](https://github.com/FleekHQ/space-daemon/pull/219/commits/9bc2f71364d4b739cf4172e80aa9ef7f3d7e3885).